src/fetcher/index.js

```diff
@@ -0,0 +1,17 @@
+import Api from '@/js/RequestApi';
+
+const api = new Api();
+const key = process.env.APP_API_KEY;
+const apiKey = '&api_key=' + key;
+const apiString = 'http://ws.audioscrobbler.com/2.0/?method=';
+const format = '&format=json';
+
+export const lastFm = ({
```

Comment:
del export and useless round brackets. lastFm - should be an instance of RaquestApi and contain all methods of this lib (class).
lastFm = new api({apiKey, ...otherParameters});